src/scripts/Update.ts

```diff
@@ -598,6 +599,47 @@ class Update implements Saveable {
             saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 32, 34); // New Mauville
             saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 84, 64); // Pledge Grove
             saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 78, 79); // Abundant Shrine
+
+            // Shards -> Gems
+            delete saveData.quests.questList;
+
+            //Setting gems = shards
+            saveData.gems = saveData.shards || {};
```

Comment:
We probably don't need this line, as if `saveData.shards` doesn't exist, the next few lines will fail when it calls `saveData.shards.shardWallet` etc.

Reply:
[b191e61](https://github.com/pokeclicker/pokeclicker/pull/1791/commits/b191e611f9850991637fe4ccd96cc1a4bea10577)